Simplenote/SPAppDelegate.m

```diff
@@ -190,6 +195,11 @@ - (void)applicationDidEnterBackground:(UIApplication *)application
 
     [self showPasscodeLockIfNecessary];
     [self cleanupScrollPositionCache];
+
+    // Schedule background refresh
+    if (@available(iOS 13.0, *)) {
+        [self.refreshManager scheduleAppRefresh];
```

Comment:
We probably should also cancel tasks if the app goes back to foreground.

Reply:
Yeah, I was thinking that, but didn't see anything in the docs that required it.  What I did notice digging into the logs yesterday is that the background service only allows for one scheduled event at a time. If you schedule a new one it removes the old one first.  But it makes sense to me to clear out any pending ones when the app enters the foreground.  Will add that today